### PR TITLE
Sections creation

### PR DESCRIPTION
**Pull request by sirfrancium**
wrapped up more work on various import scripts. Also, added startingTeacher param for the saveAllGradebooks script

### CODE REVIEW

File: api/sections/add_section_ids.py [hunk 1]
```diff
@@ -37,21 +35,54 @@ def main():
...
-                        "section_name": section_name,
-                        "student": student,
-                        "section_id": section_id}, cc_print=True)
+    if 'Student ID' in csv_sections.cols:
+        qs.logger.info('Retrieving sections from csv & matching with db...',
+                       cc_print=True)
+        for csv_section_info in qs.bar(csv_sections):
+            section_name = csv_section_info['Section Name']
+            student = csv_section_info['Student ID']
+            row_num = row_num + 1
```
Comment (br1ckb0t) [inline api/sections/add_section_ids.py]: better to do `row_num += 1`

File: api/sections/add_section_ids.py [hunk 2]
```diff
@@ -37,21 +35,54 @@ def main():
...
+    elif 'Section Code' in csv_sections.cols:
+            sections = {}
+            sections_with_id = {}
+            row_num = 1
+
+            qs.logger.info('Retrieving section code from csv...',
+                           cc_print=True)
+            for csv_section_info in csv_sections:
+                section_name = csv_section_info[u'Section Name']
+                section_code = csv_section_info[u'Section Code']
```
Comment (br1ckb0t) [inline api/sections/add_section_ids.py]: stick to either `''` or `u''`, but best not to switch. Personally, unless I'm specifically doing something unicode-specific, I just do `''`.

File: api/sections/add_section_ids.py (hunk 1 above)
Comment (sirfrancium) [inline api/sections/add_section_ids.py]: OMG THAT'S AWKWARD

File: gui/createGradebookFinalGradeFormulas.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ new QSGradebookIterator(function() {
 
     this.click("Edit it anyways");
     this.click("Add Formula");
-    $("tr.category:contains(Assignment) input").click();
+    $("tr.category:contains(Classwork) input").click();
```
Comment (br1ckb0t) [inline gui/createGradebookFinalGradeFormulas.js]: I think the default should remain at `Assignment` - seems more common to me. Thoughts?

File: api/enrollments/enroll_in_existing_sections.py [hunk 4]
```diff
@@ -11,6 +11,8 @@
 Usage: ./enroll_in_existing_sections.py {school code} {semester} filename.csv
 
 Returns: Nothing - just enrolls students in their sections.
+
+## TO DO - MATCH BY SECTION CODE
```
Comment (br1ckb0t) [inline api/enrollments/enroll_in_existing_sections.py]: My preferred syntax:
`#TODO: Something important`

File: gui/createGradebookFinalGradeFormulas.js (hunk 3 above)
Comment (sirfrancium) [inline gui/createGradebookFinalGradeFormulas.js]: Yeah... Assignment seems fine. But, I guess it could vary. Might as well leave the default there though

**Pull request merged**